pkg/controller/core/indexer/indexer.go

```diff
@@ -58,6 +60,20 @@ func IndexWorkloadClusterQueue(obj client.Object) []string {
 	return []string{string(wl.Status.Admission.ClusterQueue)}
 }
 
+func IndexLimitRangeHasContainerType(obj client.Object) []string {
```

Comment:
We might need to also check type Pod when we do validation (for min/max).

So an alternative implementation would be `IndexLimitRangeByType`. Then you can return all the types that the LimitRange has.

But we can leave this for later.